### PR TITLE
Add redirect test

### PR DESCRIPTION
**Pull request by volhadl**

### CODE REVIEW

File: cypress/integration/redirectTest.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import HomePage from '../../src/pages/HomePage'
+import * as urls from '../../testData/urls'
+
+describe("Redirect test", ()=>{
+    
+
```
Comment (KirylParkhomenka) [inline cypress/integration/redirectTest.js:6]: Please remove extra empty line.

File: cypress/integration/redirectTest.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    it("Verify URL after redirect", ()=>{  
+        HomePage.openUrl(urls.homeUrl);
+        cy.get(HomePage.mobileMenu).click()
+        HomePage.openMobileService(HomePage.selectMobileTariff)
+        cy.request('https://www.a1.by')
+
```
Comment (KirylParkhomenka) [inline cypress/integration/redirectTest.js:12]: Please remove extra empty line.

File: src/pages/HomePage.js [hunk 3]
```diff
@@ -7,6 +7,11 @@ class HomePage {
     tariffThree = '[href*="super-25"] > * > .main-hit__top > *'
     instagramButton = '[href*="instagram.com"]'
     twitterButton = '[href="https://t.me/by_mts "]'
+    mobileMenu = '[class*="burger-icon"]'
+    tariffs = '.menu-mobile__icon>[src*="icon_mobile"]'
+    mobileTariffs = '.mobile-submenu>[href="/services/mobile/"]'
+    selectMobileTariff = '.mobile-submenu > [href*="/services/mobile/tariffs/"]'
+
```
Comment (KirylParkhomenka) [inline src/pages/HomePage.js:14]: Please remove extra empty line.

File: testData/urls.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 export const homeUrl = 'https://www.mts.by'
+export const a1homeUrl = 'https://www.mts.by/'
```
Comment (KirylParkhomenka) [inline testData/urls.js:2]: Why a1 url is equal to mts url???

File: cypress/integration/redirectTest.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import * as urls from '../../testData/urls'
+
+describe("Redirect test", ()=>{
+    
+
+    it("Verify URL after redirect", ()=>{  
+        HomePage.openUrl(urls.homeUrl);
+        cy.get(HomePage.mobileMenu).click()
+        HomePage.openMobileService(HomePage.selectMobileTariff)
+        cy.request('https://www.a1.by')
```
Comment (KirylParkhomenka) [inline cypress/integration/redirectTest.js:11]: Why do you use `request` here?
You just need to check the URL after redirection.
Please review the documentation how to do it :
https://docs.cypress.io/api/commands/url.html#Assert-the-URL-is-http-localhost-8000-users-1-edit

Comment (volhadl) [reply cypress/integration/redirectTest.js:11]: 1. "HomePage.openUrl(urls.homeUrl);
        cy.get(HomePage.mobileMenu).click()
        HomePage.openMobileService(HomePage.selectMobileTariff)"
here I implemented logic for those steps
"a - открывать урл МТС
b - после этого открывать страницу с тарифами"


2. in cypress, we are not able to visit two urls in one test
There is no button to open a1

**Pull request closed without merging**